
project build/make/
diff --git a/build/make/target/product/core.mk b/build/make/target/product/core.mk
index c72bb2f26..729b8fb8e 100644
--- a/build/make/target/product/core.mk
+++ b/build/make/target/product/core.mk
@@ -50,7 +50,6 @@ PRODUCT_PACKAGES += \
     PrintSpooler \
     PrintRecommendationService \
     ProxyHandler \
-    QuickSearchBox \
     SecureElement \
     Settings \
     SettingsIntelligence \

project packages/apps/Launcher3/
diff --git a/packages/apps/Launcher3/src/com/android/launcher3/config/BaseFlags.java b/packages/apps/Launcher3/src/com/android/launcher3/config/BaseFlags.java
index f4c6380b7..1d5d97199 100644
--- a/packages/apps/Launcher3/src/com/android/launcher3/config/BaseFlags.java
+++ b/packages/apps/Launcher3/src/com/android/launcher3/config/BaseFlags.java
@@ -39,7 +39,7 @@ abstract class BaseFlags {
     public static final boolean LAUNCHER3_SPRING_ICONS = true;
 
     // Feature flag to enable moving the QSB on the 0th screen of the workspace.
-    public static final boolean QSB_ON_FIRST_SCREEN = true;
+    public static final boolean QSB_ON_FIRST_SCREEN = false;
     // When enabled the all-apps icon is not added to the hotseat.
     public static final boolean NO_ALL_APPS_ICON = true;
 
diff --git a/packages/apps/Launcher3/src/com/android/launcher3/provider/ImportDataTask.java b/packages/apps/Launcher3/src/com/android/launcher3/provider/ImportDataTask.java
index b1dd003f8..a30687d60 100644
--- a/packages/apps/Launcher3/src/com/android/launcher3/provider/ImportDataTask.java
+++ b/packages/apps/Launcher3/src/com/android/launcher3/provider/ImportDataTask.java
@@ -132,7 +132,7 @@ public class ImportDataTask {
         String profileId = Long.toString(UserManagerCompat.getInstance(mContext)
                 .getSerialNumberForUser(Process.myUserHandle()));
 
-        boolean createEmptyRowOnFirstScreen;
+        boolean createEmptyRowOnFirstScreen = false;
         if (FeatureFlags.QSB_ON_FIRST_SCREEN) {
             try (Cursor c = mContext.getContentResolver().query(mOtherFavoritesUri, null,
                     // get items on the first row of the first screen
