
project device/google/crosshatch/
diff --git a/device/google/crosshatch/aosp_blueline.mk b/device/google/crosshatch/aosp_blueline.mk
index 9a38f9fb..4f657ae0 100644
--- a/device/google/crosshatch/aosp_blueline.mk
+++ b/device/google/crosshatch/aosp_blueline.mk
@@ -30,4 +30,4 @@ PRODUCT_MANUFACTURER := Google
 PRODUCT_BRAND := Android
 PRODUCT_NAME := aosp_blueline
 PRODUCT_DEVICE := blueline
-PRODUCT_MODEL := AOSP on blueline
+PRODUCT_MODEL := Pixel 3
diff --git a/device/google/crosshatch/aosp_crosshatch.mk b/device/google/crosshatch/aosp_crosshatch.mk
index 7db148cf..a54202c4 100644
--- a/device/google/crosshatch/aosp_crosshatch.mk
+++ b/device/google/crosshatch/aosp_crosshatch.mk
@@ -30,4 +30,4 @@ PRODUCT_MANUFACTURER := Google
 PRODUCT_BRAND := Android
 PRODUCT_NAME := aosp_crosshatch
 PRODUCT_DEVICE := crosshatch
-PRODUCT_MODEL := AOSP on crosshatch
+PRODUCT_MODEL := Pixel 3 XL

project device/google/marlin/
diff --git a/device/google/marlin/aosp_marlin.mk b/device/google/marlin/aosp_marlin.mk
index 96cc5e95..1b9d9e7f 100644
--- a/device/google/marlin/aosp_marlin.mk
+++ b/device/google/marlin/aosp_marlin.mk
@@ -24,7 +24,7 @@ $(call inherit-product, $(SRC_TARGET_DIR)/product/aosp_base_telephony.mk)
 PRODUCT_NAME := aosp_marlin
 PRODUCT_DEVICE := marlin
 PRODUCT_BRAND := Android
-PRODUCT_MODEL := AOSP on msm8996
+PRODUCT_MODEL := Pixel XL
 PRODUCT_MANUFACTURER := Google
 PRODUCT_RESTRICT_VENDOR_FILES := true
 
diff --git a/device/google/marlin/aosp_sailfish.mk b/device/google/marlin/aosp_sailfish.mk
index b6e6427c..67d3e048 100644
--- a/device/google/marlin/aosp_sailfish.mk
+++ b/device/google/marlin/aosp_sailfish.mk
@@ -24,7 +24,7 @@ $(call inherit-product, $(SRC_TARGET_DIR)/product/aosp_base_telephony.mk)
 PRODUCT_NAME := aosp_sailfish
 PRODUCT_DEVICE := sailfish
 PRODUCT_BRAND := Android
-PRODUCT_MODEL := AOSP on msm8996
+PRODUCT_MODEL := Pixel
 PRODUCT_MANUFACTURER := Google
 PRODUCT_RESTRICT_VENDOR_FILES := true
 

project device/google/taimen/
diff --git a/device/google/taimen/aosp_taimen.mk b/device/google/taimen/aosp_taimen.mk
index 5653fd5..3729530 100644
--- a/device/google/taimen/aosp_taimen.mk
+++ b/device/google/taimen/aosp_taimen.mk
@@ -39,4 +39,4 @@ PRODUCT_MANUFACTURER := Google
 PRODUCT_BRAND := Android
 PRODUCT_NAME := aosp_taimen
 PRODUCT_DEVICE := taimen
-PRODUCT_MODEL := AOSP on taimen
+PRODUCT_MODEL := Pixel 2 XL
