
project build/make/
diff --git a/build/make/target/product/handheld_product.mk b/build/make/target/product/handheld_product.mk
index f504f61874..e997b8be4b 100644
--- a/build/make/target/product/handheld_product.mk
+++ b/build/make/target/product/handheld_product.mk
@@ -28,6 +28,7 @@ PRODUCT_PACKAGES += \
     Contacts \
     DeskClock \
     ExactCalculator \
+    F-DroidPrivilegedExtension \
     Gallery2 \
     LatinIME \
     Music \
