
project build/make/
diff --git a/build/make/target/product/handheld_product.mk b/build/make/target/product/handheld_product.mk
index 0d100b267..806c005bd 100644
--- a/build/make/target/product/handheld_product.mk
+++ b/build/make/target/product/handheld_product.mk
@@ -22,23 +22,23 @@ $(call inherit-product, $(SRC_TARGET_DIR)/product/media_product.mk)
 
 # /product packages
 PRODUCT_PACKAGES += \
-    Browser2 \
-    Calendar \
     Camera2 \
     Contacts \
     DeskClock \
     Email \
+    F-DroidPrivilegedExtension \
     Gallery2 \
     LatinIME \
     Launcher3QuickStep \
     Music \
     OneTimeInitializer \
     Provision \
-    QuickSearchBox \
     Settings \
     SettingsIntelligence \
     StorageManager \
     SystemUI \
+    Updater \
     WallpaperCropper \
     frameworks-base-overlays
 
