
project build/make/
diff --git a/build/make/target/product/core.mk b/build/make/target/product/core.mk
index c72bb2f26..a8d0d1850 100644
--- a/build/make/target/product/core.mk
+++ b/build/make/target/product/core.mk
@@ -37,6 +37,8 @@ PRODUCT_PACKAGES += \
     Email \
     ExactCalculator \
     ExternalStorageProvider \
+    F-Droid \
+    F-DroidPrivilegedExtension \
     FusedLocation \
     InputDevices \
     KeyChain \

project packages/apps/F-Droid/
diff --git a/packages/apps/F-Droid/Android.mk b/packages/apps/F-Droid/Android.mk
index 146b47329..9793b7a11 100644
--- a/packages/apps/F-Droid/Android.mk
+++ b/packages/apps/F-Droid/Android.mk
@@ -10,13 +10,13 @@ fdroid_root  := $(LOCAL_PATH)
 fdroid_dir   := app
 fdroid_out   := $(PWD)/$(OUT_DIR)/target/common/obj/APPS/$(LOCAL_MODULE)_intermediates
 fdroid_build := $(fdroid_root)/$(fdroid_dir)/build
-fdroid_apk   := build/outputs/apk/full/release/$(fdroid_dir)-full-release-unsigned.apk
+fdroid_apk   := build/outputs/apk/full/release/app-full-release-unsigned.apk
 
 $(fdroid_root)/$(fdroid_dir)/$(fdroid_apk):
 	rm -Rf $(fdroid_build)
 	mkdir -p $(fdroid_out)
 	ln -sf $(fdroid_out) $(fdroid_build)
-	cd $(fdroid_root)/$(fdroid_dir) && gradle assembleRelease
+	cd $(fdroid_root)/$(fdroid_dir) && ../gradlew assembleRelease
 
 LOCAL_CERTIFICATE := platform
 LOCAL_SRC_FILES := $(fdroid_dir)/$(fdroid_apk)

project packages/apps/F-DroidPrivilegedExtension/
diff --git a/packages/apps/F-DroidPrivilegedExtension/app/src/main/java/org/fdroid/fdroid/privileged/PrivilegedService.java b/packages/apps/F-DroidPrivilegedExtension/app/src/main/java/org/fdroid/fdroid/privileged/PrivilegedService.java
index 837a937..fce59b4 100644
--- a/packages/apps/F-DroidPrivilegedExtension/app/src/main/java/org/fdroid/fdroid/privileged/PrivilegedService.java
+++ b/packages/apps/F-DroidPrivilegedExtension/app/src/main/java/org/fdroid/fdroid/privileged/PrivilegedService.java
@@ -231,7 +231,7 @@ public class PrivilegedService extends Service {
                  * The client app used to set this to F-Droid, but we need it to be set to
                  * this package's package name to be able to uninstall from here.
                  */
-                pm.setInstallerPackageName(packageName, BuildConfig.APPLICATION_ID);
+                pm.setInstallerPackageName(packageName, getPackageName());
                 // Create a PendingIntent and use it to generate the IntentSender
                 Intent broadcastIntent = new Intent(BROADCAST_ACTION_UNINSTALL);
                 PendingIntent pendingIntent = PendingIntent.getBroadcast(
